web/src/main/webapp/js/functions.js

```diff
@@ -172,17 +191,26 @@ function getTestScenarios (id) {
          success: function (data) {
             if (data.length > 0) {
                 var innerHtml = "<tr class='test-scenario hidden'><td colspan=6><h4>Test Scenarios</h4><hr/>"
-                        + "<table class='table test-scenarios'><thead><tr>"
-                        + "<th>ID</th><th>Name</th><th>Status</th></tr></thead><tbody>";
+                        + "<table id='test-scenarios' class='table test-scenarios'><thead><tr>"
+                        + "<th></th><th>Name</th><th>Status</th></tr></thead><tbody>";
                 for (var i = 0; i < data.length; i++) {
-                    innerHtml += "<tr id = " + data[i].id + "><td><button class='expand-test-cases'>+</button>&nbsp;"
-                        + data[i].id + "</td><td>"
+                    innerHtml += "<tr id = " + data[i].id + "><td><button class='expand-test-cases'>+</button></td><td>"
```

Comment:
Since we already use Jquery, cant we use Jquery append here

Reply:
For jquery append, the insertion is done as the last element of the specified parent. Therefore, [jquery after](http://api.jquery.com/after/) is more appropriate for this. For clarity, the string catenation is done outside the jquery function.

Reply:
(Y)